qa/tasks/s3tests.py

```diff
@@ -228,7 +228,7 @@ def run_tests(ctx, config):
     """
     assert isinstance(config, dict)
     testdir = teuthology.get_testdir(ctx)
-    attrs = ["!fails_on_rgw", "!lifecycle"]
+    attrs = ["!fails_on_rgw", "!lifecycle_expiration"]
```

Comment:
`These are the tests which depend on timing and often end up failing`
I believe would be good to add few names of tests in commit message.

`let us disable these for now and enable them once we figure out how to set these right.`
Also can create an tracker for this.